k8s/tendermint/tendermint_container/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tendermint/tendermint:develop
+FROM tendermint/tendermint:0.13
```

Comment:
lets keep all images to unstable for now until we are ready to tag a realease?

Reply:
The tendermint we will use for k8s is still unstable, I am changing this because the current integration is based out of `0.13` and they can make breaking changes in `develop` branch as the base image.

Reply:
okay